A.java

```diff
@@ -1,5 +1,6 @@
 public class A {
     public static void main(String[] args) {
         System.out.println("Hello! How are your?");
+        System.out.println("First change");
```

Comment:
add in string 5 - System.out.println(done");

Reply:
ok, leave it

Reply:
remove strint nr 2